features/prompt_memes_a/challenge_promptmeme_setup.feature

```diff
@@ -228,7 +228,6 @@ Feature: Prompt Meme Challenge
   # TODO: We need to check the display for fandomless memes
```

Comment:
While I'm ranting about comments, TODOs are something else that tend to linger and add nothing to the repository code. For example, the ones in these challenge tests are all from 2011. (so not part of the pull request, but an illustration of why you should pretty much never commit code with TODOs in it because in general ... you won't!)